package-lock.json

```diff
@@ -0,0 +1,11188 @@
+{
```

Comment:
I suggest running `npm i` and `npm audit fix` since there's two warnings (easily fixed) and commit the `package-lock.json` file.